### PR TITLE
docs: disable mfa prompt

### PR DESCRIPTION
**Pull request by hifabienne**
### Definition of Ready

solution scenario for disabling the mfa prompt

- [x] I am happy with the code
- [x] Short description of the feature/issue is added in the pr description
- [x] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: docs/docs/guides/solution-scenarios/configurations.mdx [hunk 1]
```diff
@@ -137,3 +137,14 @@ Content-Security-Policy: frame-ancestors https://custom-domain.com
...
+
+### Disable Multi-factor (MFA) Prompt
+
+To encourage the users to more security for their accounts, a Multi-factor prompt is shown after a certain time, to allow them to configure an additional factor.
+The prompt is shown even if Multi-factor is not enforced for the users.
+
+If you do want to disable the prompt for your users, go to the login behaviour settings and set the "Multifactor Init Lifetime" to 0.
+If the setting is not configured to 0, it means that after that time, the user will be asked to setup a factor again.
+
+![Login Behaviour Settings: Multi-factor init lifetime](/img/guides/scenarios/login-settings-mfa-init-lifetime.png)
```
Comment (fforootd) [inline docs/docs/guides/solution-scenarios/configurations.mdx:149]:
```suggestion
To encourage the users to more security for their accounts, a multi-factor prompt is shown after a certain time, to prompt them to configure an additional factor.
This prompt is shown even if multi-factor is not enforced for the users.

If you do want to disable the prompt for your users, go to the login behavior settings and set the "Multifactor Init Lifetime" to 0.
If the setting is not configured to 0, it means that after that time, the user will be asked again to setup a factor.

![Login Behavior Settings: Multi-factor init lifetime](/img/guides/scenarios/login-settings-mfa-init-lifetime.png)
```

Comment (fforootd) [reply docs/docs/guides/solution-scenarios/configurations.mdx:149]: Thanks for this, one question though:

What does happen with subzero time values?

Comment (hifabienne) [reply docs/docs/guides/solution-scenarios/configurations.mdx:149]: I actually don't know what happens with subzero values, in my opinion it doesn't make sense to send them. 
Will have to check

**Comment by hifabienne** [comment]
@fforootd I quickly tested with the init lifetime, I set to -1, configuring was possible.
After that it resulted in a loop when I wanted to login, as the prompt was shown again and again. I will not document that. :)
As the behaviour is wrong, will open up an issue for fixing it.

**Review by fforootd — APPROVED** [review]

**Pull request merged**